pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1042,7 +1042,6 @@ func nodesWherePreemptionMightHelp(nodes []*framework.NodeInfo, fitErr *FitError
 		if fitErr.FilteredNodesStatuses[name].Code() == framework.UnschedulableAndUnresolvable {
 			continue
 		}
-		klog.V(3).Infof("Node %v is a potential node for preemption.", name)
```

Comment:
Instead of removing this log could it be moved to print the whole list of potential nodes after the call to `nodesWherePreemptionMightHelp` in `Preempt` (https://github.com/kubernetes/kubernetes/blob/b03069923f0bee04ed8ccb0cb49fc005c2bf24d9/pkg/scheduler/core/generic_scheduler.go#L270)?

Reply:
formatting 5k nodes in a single line? I don't know how useful this message is, perhaps just printing the number of potential nodes is enough, wdyt?

Reply:
Perhaps we can print the first K names. It might be helpful for debugging.

Reply:
level 3 seems like too much regardless

Reply:
changed to level 5 and printed a sample of maximum 10